kuma/static/js/components/user-signup/auth-modal.js

```diff
@@ -0,0 +1,36 @@
+(function() {
+    'use strict';
+
+    var authModalContainer = document.getElementById('auth-modal');
+
+    if (authModalContainer) {
+        /* The React header takes just a couple of milliseconds to
+           long to load so, we need to wrap the code below in a 
+           setTiemout */
+        setTimeout(function() {
```

Comment:
why not call this from React then?

Reply:
oh, because it is also used on non-react controlled pages?

Reply:
Exactly :+1: